app/app.go

```diff
@@ -353,7 +352,7 @@ func New(
 		// cms must be overridden before the other options, because they may use the cms,
 		// FIXME we are assuming the cms won't be overridden by the other options, but we can't be sure.
 		cms := rootmulti.NewStore(filepath.Join(homePath, "data", "memiavl.db"), logger)
-		cms.SetAsyncWAL(cast.ToBool(appOpts.Get(FlagAsyncWAL)))
+		cms.SetAsyncWAL(cast.ToBool(appOpts.Get(FlagAsyncCommit)))
```

Comment:
not sure why set SetAsyncWAL with FlagAsyncCommit

Reply:
Good catch, I'll change